P0009/P0009.bs

```diff
@@ -1744,50 +1744,76 @@ template<class OtherExtents>
 <br/>
 <br/>
 
-<b>26.7.� Accessor [mdspan.accessor]</b>
+<b>26.7.� Accessor Policy [mdspan.accessor]</b>
 
-<br/>
-<b>26.7.�.1 Accessor requirements [mdspan.accessor.reqs]</b>
+An *accessor policy* defines types and operations by which
+a contiguous set of objects of type *ElementType* are accessed. 
 
-1. An *accessor* is a class that converts a pointer and an offset into a reference. *[Note:* The intended semantic is that the reference refers to a value at the given offset from the given pointer. *—end note]*
+<br/>
+<b>26.7.�.1 Accessor policy requirements [mdspan.accessor.reqs]</b>
 
-5. An *accessor* shall meet the requirements of `DefaultConstructible`, `CopyAssignable`, and the requirements in table �.
+An accessor policy defines an accessor type and operations for applying
+subscripting-like [expr.sub],
+conversion to pointer [conv.array],
+and offset operations to an accessor.
+The subscript operation returns an object which provides access to the indexed element,
+this return type may not be `T&`.
+The offset operation returns an accessor for the contiguous subset of elements beginning at the offste value,
+this return type may not be the same accessor type..
+An accessor constructor may impose additional restrictions on the 
+contiguous set of objects; e.g., restrict `T` to be trivially copyable or have a wider alignment.
 
 In Table �:
-  * `A` denotes an accessor.
-  * `a` denotes a value of type `A`.
-  * `p` denotes a value of type `A::pointer`.
-  * `i` denotes an integer.
+  * `A` denotes an accessor policy.
+  * `a` denotes an object of type `A`.
+  * `p` denotes an object of type `typename A::pointer`.
+  * `i` denotes a `ptrdiff_t` value.
 
-Table �: Accessor requirements
+Table �: Accessor policy requirements
 <table border=1>
 <tr>
   <th>Expression</th>
   <th>Return Type</th>
   <th>Requirements/Notes</th>
 </tr>
 <tr>
-  <td>`A::value_type`</td>
-  <td></td>
+  <td>`A::element_type`</td>
   <td></td>
+  <td>Type of elements accessed</td>
 </tr>
 <tr>
   <td>`A::pointer`</td>
   <td></td>
-  <td>*Requires:* `A::pointer` shall meet the requirements of random access iterator (**[random.access.iterators]**), and `iterator_traits<A::pointer>::value_type` shall be exactly `A::value_type`.</td>
+  <td>Type through which the contiguous set of elements are accessed. *Requires:* Is DefaultConstructible, CopyConstructible, and CopyAssignable </td>
 </tr>
 <tr>
   <td>`A::reference`</td>
   <td></td>
-  <td>*Requires:* `iterator_traits<A::pointer>::reference` shall be convertible to `A::reference` </td>
+  <td>Type through which an element is accessed</td>
+</tr>
+<tr>
+  <td>`A::offset_policy`</td>
+  <td></td>
+  <td>Accessor policy compatible with returning a offseted pointer.</td>
+</tr>
+<tr>
+  <td>`A::decay(p)`</td>
+  <td>`A::element_type*`</td>
+  <td>*Returns:* A standard pointer to the contiguous set of objects accessed by `p`.</td>
```

Comment:
Yeah we had some discussion about the span function should always exists. Dan also thinks that based on the discussions in LEWG they might have issues if we remove the capability of getting a span out of this.

Reply:
Should just be *Returns:  A pointer [dcl.ptr] to ..."
Citation to avoid confusion between `AccessorPolicy::pointer` (unless it had a different name such as `accessor`) and `[dcl.ptr]`.